docs/12-self-hosting/03-host-creation/02-multipass.mdx

```diff
@@ -0,0 +1,152 @@
+---
+toc_max_heading_level: 4
+---
+
+# VMs with Multipass (Basic)
+
+Multipass is a light-weight Virtual Machine Manager for Linux, Windows and MacOS. It's designed for
+developers who want to quickly create a fresh Ubuntu environment with a single command. It uses the
+native hypervisor for whichever platform it is installed on (KVM on Linux, Hyper-V on Windows and
+HyperKit on MacOS) to run VMs with minimal overhead. It can also use VirtualBox on Windows and
+MacOS. The biggest limitation of Multipass is that it only creates Ubuntu VMs.
+
+- [Official Website](https://multipass.run/)
+- [Official Github Repo](https://github.com/canonical/multipass)
+
+## Installation
+
+To install multipass on Linux use the commands below.
+
+```bash
+sudo apt-get install snapd
+sudo snap install core
+sudo snap install multipass
+```
+
+For installation on Windows and MacOS, refer to the official installation instructions:
+
+- [How to install Multipass on Windows](https://multipass.run/docs/installing-on-windows)
+- [How to install Multipass on MacOS](https://multipass.run/docs/installing-on-macos)
+
+## Creating a VM
+
+- Set values
+
+  ```bash
+  # The name of the Virtual Machine
+  export VM_NAME="gameci"
+
+  # The name of the user to create
+  export VM_USER="vmadmin"
+
+  # Number of CPU cores to allocate to the VM
+  export VM_CPUS="2"
+
+  # Amount of Disk Space to allocate to the VM.
+  # Cannot exceed available on host.
+  export VM_DISK="32G"
+
+  # Amount of RAM to allocate to the VM.
+  # Cannot exceed available RAM on host.
+  export VM_MEM="8G"
+
+  # Set path on MacOS systems
+  export PATH="$PATH:/usr/local/bin/multipass"
+
+  # Set path on Linux system
+  export PATH="$PATH:/usr/local/bin/multipass"
+  ```
+
+- Create a password
+
+  ```bash
+  # Install the mkpasswd utility
+  sudo apt install -y whois
+
+  read PW_STRING
+  export PASSWORD=$(mkpasswd -m sha-512 --rounds=4096 "$PW_STRING" -s "saltsaltlettuce")
+  ```
+
+- Create an ssh-key for authenticating with the VM
+
+  ```bash
+  ssh-keygen -C $VM_USER -f runner
+  ```
```

Comment:
Might be helpful to comment something about security implications here concenring password generation and ssh key generation. Something like 

Note on Security: For password and SSH key generation, always use strong, unique passwords, and safeguard SSH private keys.